src/query/functions-v2/tests/it/scalars/testdata/array.txt

```diff
@@ -48,7 +48,7 @@ raw expr       : length(array())
 checked expr   : length<Array(Nothing)>(array<>())
 optimized expr : 0_u8
 output type    : UInt8
-output domain  : Unknown
```

Comment:
Can you add a case for `a = 888`?

Reply:
This case is just for an empty array, it will always return zero.
We already have some cases for no empty array.